platform/platformFilePaths.cmake

```diff
@@ -0,0 +1,12 @@
+# This file is to add source files and include directories
+# into variables so that it can be reused from different repositories
+# in their Cmake based build system by including this file.
+
+# PLATFORM library source files.
+set( PLATFORM_SOURCES
+     "${PLATFORM_DIR}/posix/src/reconnect_posix.c" )
+
+# PLATFORM library Public Include directories.
+set( PLATFORM_INCLUDE_PUBLIC_DIRS
+     "${PLATFORM_DIR}/include"
+     "${PLATFORM_DIR}/*/include" )
```

Comment:
Overall, looks good but not sure if we wanna include all header files from `${PLATFORM_DIR}/*/include`. For example, we probably don't want to add POSIX includes for AFR. Maybe we can have separate `CMakeLists.txt` for each subdirectory and possibly separate target for each that is selectively linked to main platform target.

Reply:
Yes, this was make shift cmake file. During implementation we can change it as needed.